modules/delegates.js

```diff
@@ -295,25 +385,23 @@ __private.checkDelegates = function (publicKey, votes, state, cb) {
  * @returns {setImmediateCallback} 
  */
 __private.loadDelegates = function (cb) {
-	var secrets;
 
-	if (library.config.forging.secret) {
-		if (Array.isArray(library.config.forging.secret)) {
-			secrets = library.config.forging.secret;
-		} else {
-			secrets = [library.config.forging.secret];
-		}
-	}
+	var secretsList = library.config.forging.secret;
 
-	if (!secrets || !secrets.length) {
+	if (!secretsList || !secretsList.length || !library.config.forging.force || !library.config.forging.defaultKey) {
```

Comment:
Thanks for clarification Isabello. So, this looks to me the right condition. It will enable forging on delegates for devnet but for the rest of *nets, we'd need to hit the `/api/delegate/forging/enable` endpoint.